.github/workflows/python-package.yml

```diff
@@ -0,0 +1,30 @@
+name: Python package
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.5]
```

Comment:
FTR Python versions up to 3.9 and PyPA are also natively supported on GHA.
Also, there's a possibility to use less conventional builds from deadsnakes. Example: https://github.com/pypa/setuptools/blob/df9157c/.github/workflows/python-tests.yml#L66.